Readme.md

```diff
@@ -179,285 +179,13 @@ kubi explain <another_token> for explaining another token
-  ```
-
-2. And edit your `kubeadm-config.yml` file with the following values:
-
-  ```yaml
-  extraArgs:
-    authentication-token-webhook-config-file: /etc/kubernetes/additionnals/webhook.yml
-  extraVolumes:
-    - name: additionnals
-      hostPath: /etc/kubernetes/additionnals
-      mountPath: /etc/kubernetes/additionnals
-  ```
-
-3. Copy the webhook file to `/etc/kubernetes/additionnals` folder.
-
-# Roadmap
-
-The following features should be available soon.
-
-- Allow usage of static mapping ( a json file mapping with LDAP group and Kubernetes namespaces)
-- Expose /metrics
-
-
-# Development environment
-
-You can easily contribute to this project by using a development environment, follow the installation step from [Installation](#installation) until the [Deploy the prerequisites](#deploy-the-prerequisites).
-
-## Deploy the local config
-
-```bash
-kubectl apply -f https://raw.githubusercontent.com/ca-gip/kubi/master/deployments/kube-local-config.yml
-```
-
-## Copy the secret from you Kubernetes cluster
-
-Create secret dirs on your local machine
-
-```bash
-mkdir -p /var/run/secrets/{certs,ecdsa,kubernetes.io}
-```
-
-The mapping between the secret:key and file path
-
-| Secret Name | Secret Key | Local Path |
-|-------------|------------|------------|
-| kubi-user-<hash> | ca.crt | /var/run/secrets/kubernetes.io/serviceaccount/ca.crt |
-| kubi-user-<hash> | token | /var/run/secrets/kubernetes.io/serviceaccount/token |
-| kubi | tls.crt |/var/run/secrets/certs/tls.crt |
-| kubi | tls.key | /var/run/secrets/certs/tls.key |
-| kubi-encryption-secret | ecdsa-key.pem | /var/run/secrets/ecdsa/ecdsa-key.pem |
-| kubi-encryption-secret | ecdsa-public.pem | /var/run/secrets/ecdsa/ecdsa-public.pem |
-
-You can execute the following commands to gather all the required secrets then decode and save them
-```bash
-kubectl -n kube-system get secrets $(kubectl -n kube-system get sa kubi-user -o "jsonpath={.secrets[0].name}") -o "jsonpath={.data['ca\.crt']}" | base64 -d > /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-kubectl -n kube-system get secrets $(kubectl -n kube-system get sa kubi-user -o "jsonpath={.secrets[0].name}") -o "jsonpath={.data['token']}" | base64 -d > /var/run/secrets/kubernetes.io/serviceaccount/token
-kubectl -n kube-system get secrets kubi -o "jsonpath={.data['tls\.crt']}" | base64 -d > /var/run/secrets/certs/tls.crt
-kubectl -n kube-system get secrets kubi -o "jsonpath={.data['tls\.key']}" | base64 -d > /var/run/secrets/certs/tls.key
-kubectl -n kube-system get secrets kubi-encryption-secret -o "jsonpath={.data['ecdsa-key\.pem']}" | base64 -d > /var/run/secrets/ecdsa/ecdsa-key.pem
-kubectl -n kube-system get secrets kubi-encryption-secret -o "jsonpath={.data['ecdsa-public\.pem']}" | base64 -d > /var/run/secrets/ecdsa/ecdsa-public.pem
-```
-
-
-## Running
-
-At the base of this project execute the go run with the required variable
+- [Contributing to kubi](https://github.com/amiraaz/kubi/blob/master/CONTRIBUTING.md).
```

Comment:
Wrong link

Reply:
sorry but i don't understand why it's wrong ?